packages/hierarchies/src/hierarchies/internal/operators/Grouping.ts

```diff
@@ -62,6 +63,7 @@ export function createGroupingOperator(
         );
       }),
       concatAll(),
+      releaseMainThreadOn(100),
```

Comment:
why are we using a smaller number here compared to the one in `TreeQueryResultsReader`?

Reply:
This is an optimal value that I determined through experimentation. When I use 1000 here, p95 of blocks increases from 30 ms to 135 ms on my machine. It seems like we are doing much more work for the postprocessed nodes than for those at the beginning.

Reply:
okay, let's keep it this way for now and look into making more micro-optimizations